compose/testnets/karknu_10.2.0_anthithesis/README.md

```diff
@@ -0,0 +1,13 @@
+# Description
+
+https://github.com/IntersectMBO/ouroboros-network/issues/5058
```

Comment:
Would be great to add:
1. a high-level description of the bug (as we discussed in slack)
2. some kind of assessment/experience report on how this test fared in AT

Reply:
I will add a description of this bug later.